client/components/sizes.jsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import Inventory from './inventory.jsx';
+
+var Sizes = (props) => {
+
+  return (
+    <div class="inventory">
+    {props.styleDetails.stock ? props.styleDetails.stock.map((stock, index) => {
+        return <Inventory size={stock.size} count={stock.count} key={index}/>
+    }): null}
+  </div>
+  )
+}
+
```

Comment:
Good looking Sizes component.
Only feedback I have is the similar feedback I provided to your Inventory component about destructuring your props if you find that helpful!